audbackend/core/utils.py

```diff
@@ -9,6 +11,40 @@
 BACKEND_ALLOWED_CHARS_COMPILED = re.compile(BACKEND_ALLOWED_CHARS)
 
 
+def check_path_and_ext(
```

Comment:
Instead of converting the virtual backend path directly, we only call this function in the interface. It applies a couple of small checks, see list below.